dotCMS/src/main/java/com/dotmarketing/business/ajax/RoleAjax.java

```diff
@@ -71,6 +76,8 @@
 
 public class RoleAjax {
 
+	private static final SystemEventsAPI systemEventsAPI = APILocator.getSystemEventsAPI();
```

Comment:
this should be init on the constructors, with the test visible pattern we follow, and not sure why it is static.